deploy/buildtools/buildconfig.py

```diff
@@ -94,6 +98,27 @@ def get_chisel_triplet(self) -> str:
         """
         return f"{self.DESIGN}-{self.TARGET_CONFIG}-{self.PLATFORM_CONFIG}"
 
+    def get_frequency(self) -> int:
+        """Get the build-specific frequency config.
+
+        Returns:
+            specified frequency (int)
+        """
+        if self.frequency is None:
```

Comment:
I've disambiguated with the naming of the getter (to `get_f1_frequency`), and checked the platform earlier in the __init__ so it fails faster rather than later on in the flow when the getter is used.